app/presenters/contributions/delete.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Contributions
+  class Delete < ApplicationPresenter
+
+    def content
+      mustache[:content] ||= begin
+        {
+          title: t('title') + ' - ' + @contribution.dc_title.join('; '),
+          confirmation_text: [t('delete_1', scope: 'contribute.contributions.confirm'), t('delete_2', scope: 'contribute.contributions.confirm')]
```

Comment:
Hovering over the cancel button gets a missing translation error.

Reply:
added a string to localeapp